Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -117,7 +117,7 @@ function InstallCUDADrivers() {
     ;;
     esac
 
-    cp /var/lib/dkms/nvidia/*/log/make.log ${HOME}/nvidia_dkms_make.log
+    find /var/lib/dkms/nvidia* -name make.log -exec cp {} ${HOME}/nvidia_dkms_make.log \;
```

Comment:
the current test failure is given that the cp will fail as the log is in a different path.
What I was thinking in the issue was that we have to somehow skip the exit code for the log find/copy here.

So even presumable that the log will not be found, the test will continue to report pass/fail based on the current checks in the script, and not incorrectly fail because of the exit code from the cp command.

Reply:
Then what's the purpose for #269, if make.log will exist after install cuda driver, then we can't skip it, otherwise we can't.

Reply:
the log is to be copied so we have it archived in case of compile errors.
What PR #269 didn't have was that if that copy of the make log will fail, the entire test will be reported as failed. And that's not what we want, having the log is optional and should not fail the test.